Tools/wasm/.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*.{html,js}]
+indent_size = 4
```

Comment:
i'm not sure that file is required

Reply:
You're right that it's not strictly required, but CPython already has a root `.editorconfig` file which informs text editors of the indentation and whitespace style to use. The [root file](https://github.com/python/cpython/blob/main/.editorconfig) doesn't have rules for HTML and JS files, which I noticed because my editor had a default setup that was inserting 2 spaces when I hit tab in these files.

I'm not sure whether the other `*.js` and `*.html` files in this repository use 4 spaces consistently, so I made a local `.editorconfig` file instead of adding to the root one.